maps/submaps/surface_submaps/mountains/mountains.dm

```diff
@@ -41,6 +41,8 @@
 #include "Cliff1.dmm"
 #include "excavation1.dmm"
 #include "spatial_anomaly.dmm"
+#include "speakeasy_vr.dmm"
+#include "Campsite_vr.dmm"
```

Comment:
I think you added this by accident?

Reply:
yes, that was a result of me using upload to make a pull, I've fixed the issue.